### PR TITLE
Add route parameters

### PR DESCRIPTION
**Pull request by KentoMoriwaki**
This PR enables to generate routing parameters, which is required to use the generated client.

This depends on https://github.com/ngtk/open_api/pull/7

### CODE REVIEW

**Comment by KentoMoriwaki** [comment]
@ngtk plz review after https://github.com/ngtk/open_api/pull/7

File: lib/open_api_annotator/paths_builder.rb [hunk 1]
```diff
@@ -81,38 +93,43 @@ def resolve_type(controller_name, action_name)
...
         action = route.requirements[:action]
-        Route.new(http_verb: route.verb, path: path, controller_name: controller, action_name: action)
+        Route.new(http_verb: route.verb, path: path, controller_name: controller, action_name: action, parameters: parameters)
       end
     end
   end
 
   class PathResolver
-    def resolve(ast)
+    def resolve(ast, parameters = [])
```
Comment (ngtk) [inline lib/open_api_annotator/paths_builder.rb]: I think it is a little bit hard to suppose what will be done in this method or its name. So, you need to separate the method.

**Review by ngtk — CHANGES_REQUESTED** [review]
Thank you for pull-requesting! 😄 
It seems fine roughly.  but, I have 1 comment and you need to add the tests with the nominal scenarios at least.

File: lib/open_api_annotator/paths_builder.rb (hunk 1 above)
Comment (KentoMoriwaki) [reply lib/open_api_annotator/paths_builder.rb]: I renamed this param to `parameters_context` and added a test.
Isn't it clear enough?

Comment (ngtk) [reply lib/open_api_annotator/paths_builder.rb]: better. I can aprove this.

**Review by ngtk — APPROVED** [review]
LGTM 👍 Thanks!

**Comment by ngtk** [comment]
Could you update open_api to v0.4.0 in this PR to include ngtk/open_api#7?

**Comment by ngtk** [comment]
Thank you! 😊

**Pull request merged**